html-aam/index.html

```diff
@@ -3225,6 +3225,59 @@ <h4 id="el-input-checkbox">`input` <span class="el-context">(`type` attribute in
             </tr>
           </tbody>
         </table>
+        <h4 id="el-input-checkbox-switch">`input` <span class="el-context">(`type` attribute in the Checkbox state)</span> with the <code>switch</code> attribute</h4>
+        <table aria-labelledby="el-input-checkbox-switch">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>
+                <a data-cite="html">`input`</a>
+                <span class="el-context"
+                  >(<a data-cite="html/input.html#attr-input-type">`type`</a> attribute in the <a data-cite="html/input.html#checkbox-state-(type=checkbox)">Checkbox</a> state)</span
+                >
+                with the <a data-cite="html/input.html#attr-input-switch">`switch`</a> attribute
+              </td>
+            </tr>
+            <tr>
+              <th>[[wai-aria-1.2]]</th>
+              <td>
+                <a class="core-mapping" href="#role-map-checkbox">`switch`</a> role, with the <a class="core-mapping" href="#ariaCheckedMixed">`aria-checked`</a> state set to "true" if the element's
+                <a data-cite="html/form-control-infrastructure.html#concept-fe-checked">checkedness</a> is true, or "false" otherwise
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
```

Comment:
The full name is the "The macOS Accessibility Protocol" or "AX API" for short. I don't recall this even being shortened to just "AX", and in either case, the NSAccessibility link is just one subset portion. This might be the best link in general. https://developer.apple.com/documentation/accessibility Should this be changed elsewhere? HAppy to discuss offline since this may be OT for this PR.

Reply:
From what I can see looking back to some of the original PRs, this spec has always referred to it as just "AX".  i notice though that it is not referred to consistently across AAMs

in the accessibility api sections, HTML AAM refers to it as macOS Accessibility Protocol [AXAPI].  

in other many other specs that list out the full name (often in api / important term sections), it's referred to by it's older name - The [Mac OS X Accessibility Protocol](https://developer.apple.com/documentation/appkit/accessibility/nsaccessibility) [[AXAPI](https://www.w3.org/TR/accname-1.1/#bib-axapi)] - if it's even directly mentioned at all.


in the mapping tables, it's handled like:
dpub AAM - Mac AX API
html aam - AX
core aam, mathml aam - AX API
graphics aam - AXAPI


i have another issue this can be addressed with, at least for html aam

Reply:
@scottaohara Would you like me to do raise a separate issue/PR for updating this throughout html-aam?